apps/functions/applications-migration/common/migrators/nsip-project-migration.js

```diff
@@ -59,22 +59,13 @@ const getNsipProjects = async (log, caseReference) => {
 	log.info(`Retrieved projects ${JSON.stringify(projects)}`);
 
 	return projects.map((project) => ({
-		dateIAPIDue: null,
-		jRPeriodEndDate: null,
-		operationsLeadId: null,
-		operationsManagerId: null,
-		caseManagerId: null,
-		nsipOfficerIds: [],
-		nsipAdministrationOfficerIds: [],
-		leadInspectorId: null,
-		inspectorIds: [],
-		environmentalServicesOfficerId: null,
-		legalOfficerId: null,
-		applicantId: null,
-		migrationStatus: null,
-		dateOfReOpenRelevantRepresentationStart: null,
-		dateOfReOpenRelevantRepresentationClose: null,
 		...project,
-		regions: project.region ? [project.region] : []
+		nsipOfficerIds: valueToArray(project.nsipOfficerIds),
+		nsipAdministrationOfficerIds: valueToArray(project.nsipAdministrationOfficerIds),
+		inspectorIds: valueToArray(project.inspectorIds),
+		migrationStatus: project.migrationStatus ? Boolean(project.migrationStatus) : null,
+		regions: valueToArray(project.region)
 	}));
 };
+
+const valueToArray = (value) => (value ? [value] : []);
```

Comment:
Is it worth checking if value is an array? If ODW were to fix this issue, and return an array, then this would break.

Reply:
We should only ever get a string or null for the values this function is being used for. The data is the output of the SQL query so don't think we'll get an array, just a single value per row

Reply:
Hmm OK, I guess this is the downside of using the curated layer instead of service bus/an API - we're not _quite_ dealing with schema-compliant formats.